### DISCUSSION TITLE
MongoDb with prisma showing error

### DISCUSSION THREAD

**Discussion by bytesagar** [prisma/prisma#9152] [Q&A] [answered]
``` Invalid `prisma.user.create()` invocation:

  Error in connector: Database error. error code: unknown, error message: Transactions are not supported by this deployment ```

I am getting this error.

**Comment by ArthurGerbelot** [comment, marked as answer]
I had the same issue and found [on the doc ](https://www.prisma.io/docs/concepts/database-connectors/mongodb#troubleshooting) that we try to use Prisma's MongoDB connector on a deployment that has no replica set configured.

**Reply by ryands17** [reply]
Yes you need to use a deployment that has a replica set configured.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
